packages/design-system-docs/src/pages/components/Button/button.example.html

```diff
@@ -130,3 +83,54 @@ <h6 class="preview__label">Size variant buttons</h6>
 <button class="ds-c-button ds-c-button--big">
   Big
 </button>
+
+<h6 class="preview__label">Inverse buttons</h6>
```

Comment:
@line47 In this PR I combined the 2 inverse examples, does that sound ok to you?